rq_dashboard/templates/rq_dashboard/base.html

```diff
@@ -18,13 +18,25 @@
 
 </div>
 
+<div id="confirmation-modal" class="modal hide fade">
+  <div class="modal-header">
+    <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+    <h3>Confirm the action</h3>
```

Comment:
I think It could be better to use something like 

    Do you really want to {{ action }}?

as a text here and provide action name when calling `showModal` function.
Does It sound reasonable?

Reply:
I'll try, do you have any tool to interpolate the messages in javascript? I mean, it works like angular so I can add a variable in "window" scope for example, and then it will be interpolated there?

Reply:
Currently It's implemented as [text/template script blocks](https://github.com/eoranged/rq-dashboard/blob/master/rq_dashboard/templates/rq_dashboard/dashboard.html#L43).
But since It's not MVVM, I'm not sure If It will work the way we want It to.

The easiest way is to create element with id like 
```
<span id="confirmationModalAction"></span>
```
And simply modify Its contents with jQuery.

I'm currently thinking about changing JS framework to React/Vue, which should help with many interface issues, so hacks like this are no problem as long as they are simple and straightforward.